### PR TITLE
Update organizations.rst

### PR DESCRIPTION
**Pull request by cyril-liu**

### CODE REVIEW

**Comment by stsewd** [comment]
Hi @cyril-liu, could you explain what this PR fix? Thanks

File: docs/business/organizations.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Organizations
 -------------
 
+Organizations 2
```
Comment (RichardLitt) [inline docs/business/organizations.rst]: I'm confused; is there a need for another heading?

**Comment by agjohnson** [comment]
Going to close this PR. We get PRs like this a lot on our template project, which are normally spam, but it looks like @cyril-liu might just be testing things out :)

**Comment by cyril-liu** [comment]
yes, I'm just a new comer to GitHub. Sorry for all the inconvenience caused!

**Comment by stsewd** [comment]
Thanks for the clarification, nothing to worry about

**Pull request closed without merging**